scripts/common/prompts.sh

```diff
@@ -341,18 +289,14 @@ function prompt_for_private_ip() {
         return
     fi
 
-    if ! prompts_can_prompt ; then
-        bail "Multiple network interfaces present, please select an IP address. Try passing the selected address to this script e.g. 'sudo ./install.sh private-address=1.2.3.4' or assign an IP address to the privateAddress field in the kurl add-on."
```

Comment:
i do not understand why this is changing

Reply:
Because it would only occurs if `prompts_can_prompt ` which was removed.